meteor-app/imports/ui/pages/workspace-charts/component.js

```diff
@@ -19,13 +20,16 @@ export default class ChartsPage extends React.Component {
     rangeMax: PropTypes.number.isRequired,
 
     // Callback function for updating filter values
-    updateFilterMin: PropTypes.func.isRequired,
-    updateFilterMax: PropTypes.func.isRequired,
+    updateFilter: PropTypes.func.isRequired,
+
+    // Make sure the filter value is correct.
+    checkFilterValue: PropTypes.func.isRequired,
```

Comment:
If `checkFilterValue` is a generic helper function, (and to me it seems to be except the name should be `clamp`, ) then the view should directly import it. It doesn't make sense to me to have it as part of the interface of the component.